api/src/main/java/org/openmrs/api/CohortService.java

```diff
@@ -193,5 +194,38 @@
 	 */
 	@Authorized( { PrivilegeConstants.GET_PATIENT_COHORTS })
 	public Cohort getCohortByUuid(String uuid);
-	
+
+	/**
+	 * Adds membership to a Cohort
+	 *
+	 * @param cohort
+	 * @param cohortMembership
+	 * @return the Cohort that was passed in with the CohortMembership
+	 * @should add Membership to Cohort and save the Cohort
+	 * @throws APIException
+	 */
+	@Authorized( {PrivilegeConstants.EDIT_COHORTS })
+	public Cohort addMembershipToCohort(Cohort cohort, CohortMembership cohortMembership) throws APIException;
```

Comment:
I know we have this throws APIException on our existing service methods but it's redundant because it's a runtime exception, you can get rid of it, same for the public modifier, it is redundant on interfaces